src/Auth/Database/AdminTablesSeeder.php

```diff
@@ -67,63 +72,73 @@ public function run()
             ],
         ]);
 
-        Role::first()->permissions()->save(Permission::first());
+        $roleModel::first()->permissions()->save($permissionModel::first());
 
         // add default menus.
-        Menu::truncate();
-        Menu::insert([
-            [
-                'parent_id' => 0,
-                'order'     => 1,
-                'title'     => 'Dashboard',
-                'icon'      => 'fa-bar-chart',
-                'uri'       => '/',
-            ],
-            [
-                'parent_id' => 0,
-                'order'     => 2,
-                'title'     => 'Admin',
-                'icon'      => 'fa-tasks',
-                'uri'       => '',
-            ],
-            [
-                'parent_id' => 2,
-                'order'     => 3,
-                'title'     => 'Users',
-                'icon'      => 'fa-users',
-                'uri'       => 'auth/users',
-            ],
-            [
-                'parent_id' => 2,
-                'order'     => 4,
-                'title'     => 'Roles',
-                'icon'      => 'fa-user',
-                'uri'       => 'auth/roles',
-            ],
-            [
-                'parent_id' => 2,
-                'order'     => 5,
-                'title'     => 'Permission',
-                'icon'      => 'fa-ban',
-                'uri'       => 'auth/permissions',
-            ],
-            [
-                'parent_id' => 2,
-                'order'     => 6,
-                'title'     => 'Menu',
-                'icon'      => 'fa-bars',
-                'uri'       => 'auth/menu',
-            ],
-            [
-                'parent_id' => 2,
-                'order'     => 7,
-                'title'     => 'Operation log',
-                'icon'      => 'fa-history',
-                'uri'       => 'auth/logs',
-            ],
+        $menuModel::truncate();
+
+        $menu[0] = $menuModel::create([
+            'order'     => 1,
+            'title'     => 'Dashboard',
+            'icon'      => 'fa-bar-chart',
+            'uri'       => '/',
+        ]);
+        $menu[0]->save();
+
+        $menu[1] = $menuModel::create([
+            'order'     => 2,
+            'title'     => 'Admin',
+            'icon'      => 'fa-tasks',
+            'uri'       => '',
+        ]);
+        $menu[1]->parent()->associate($menu[0]);
```

Comment:
The primary key is not always a sequence of ids. It can be an UUID or other generated id types. Using Eloquent helpers to be sure to get the correct ID.